Makefile

```diff
@@ -31,7 +31,7 @@ SRC_FILES := $(wildcard *.cpp) $(wildcard *.c) nasm-utils/nasm-utils-helper.c
 SRC_FILES := $(filter-out $(PFC_SRC), $(SRC_FILES))
 
 ifeq ($(USE_LIBPFC),1)
-LDFLAGS += -Llibpfc '-Wl,-rpath=$$ORIGIN/libpfc/' -L$(PFM_LIBDIR) '-Wl,-rpath=$$ORIGIN/$(PFM_LIBDIR)/'
+LDFLAGS += -Llibpfc '-Wl,-rpath=$$ORIGIN/libpfc/' -no-pie -L$(PFM_LIBDIR) '-Wl,-rpath=$$ORIGIN/$(PFM_LIBDIR)/'
```

Comment:
Why is this inside the `USE_LIBPFC == 1` case? Shouldn't it always be added to `LDFLAGS`?

Reply:
Sorry, I was only testing that case. But you are right `-no-pie` should be global flag.